tests_v2/test_ctrl_enter_keybind.py

```
@@ -24,17 +25,40 @@
       - /tmp/cmux-debug.sock          -> "cmux DEV"
       - /tmp/cmux-debug-foo.sock      -> "cmux DEV foo"
-      - ~/Library/Application Support/cmux/cmux.sock -> "cmux"
+      - ~/Library/Application Support/cmux/com.cmuxterm.app.sock -> "cmux"
       - /tmp/cmux-foo.sock            -> "cmux foo"
     """
     base = Path(socket_path).name
+    env_tag = sanitize_tag_slug(os.environ.get("CMUX_TAG") or "")
+    if env_tag and (
+        base == "com.cmuxterm.app.dev.sock"
+        or base.startswith("com.cmuxterm.app.dev.")
+        or base.startswith("cmux-debug")
```

Ruff (0.15.13)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)